packages/app/src/app/pages/Dashboard/Components/Repository/RepositoryListItem.tsx

```diff
@@ -27,7 +27,6 @@ export const RepositoryListItem: React.FC<RepositoryProps> = ({
     <ListAction
       align="center"
       css={css({
-        paddingX: 0,
```

Comment:
This padding is needed otherwise the boundaries of the item aren't clickable:

![Kapture 2023-02-01 at 22 27 30](https://user-images.githubusercontent.com/24959348/216208556-2d4ae7bd-4627-4792-8796-e17b4839b850.gif)